netbox/netbox/views/generic/object_views.py

```diff
@@ -399,6 +405,7 @@ def post(self, request, *args, **kwargs):
 
                     # If cloning is supported, pre-populate a new instance of the form
                     params = prepare_cloned_fields(obj)
+                    params = self.get_extra_addanother_params(request, params)
```

Comment:
Rather than passing `params` for each call, we can just update `params` with the output of `get_extra_addanother_params()`:

```suggestion
                    params.update(self.get_extra_addanother_params(request))
```